lib/rules/font-family-no-missing-generic-family-keyword/index.js

```diff
@@ -16,7 +16,7 @@ const messages = ruleMessages(ruleName, {
 const isFamilyNameKeyword = (node) =>
 	!node.quote && keywordSets.fontFamilyKeywords.has(node.value.toLowerCase());
 
-function rule(actual) {
+function rule(actual, options = {}) {
```

Comment:
Instead of using a default parameter, we should update the parameters of the `validateOptions` function below:

```js
const validOptions = validateOptions(
			result,
			ruleName,
			{ actual },
			{
				actual: options,
				possible: {
					ignoreFontFamilies: [_.isString, _.isRegExp]
				},
				optional: true,
			},
		);
```